e2e/app.spec.ts

```diff
@@ -148,21 +168,81 @@ describe('App', function() {
+    expect(actionDetailsText).to.have.string(actions[ 0 ].get('description'));
+
+    /**
+     * 7. Clicking on the theme on the action page, and thus displaying the correct Theme page.
+     */
+
+    // Click on the theme
+    const actionPageThemeTitleFinder = await actionPage.getThemeTitle();
+    await expect(actionPageThemeTitleFinder.isPresent()).to.eventually.equal(true);
+    await browser.wait(EC.elementToBeClickable(actionPageThemeTitleFinder), 5000);
+
+    // Click on the theme title
+    actionPageThemeTitleFinder.click();
+    // OK... for an unknown reason, without this 1ms of sleep, the click does not perform the transition to the Theme page...
+    // It justs stalls on the Action page without doing nothing.
+    await browser.sleep(1);
+
+    // Wait for the Theme page to show up ono the DOM
+    const themePageFinder = await themePage.getPage();
+    await presenceOf(themePageFinder);
+    await expectDisplayed(themePageFinder, { elementName: 'Theme Page' });
+    await invisibilityOf(actionPageFinder);
+    await expectDisplayed(actionPageFinder, { elementName: 'Action Page', shouldBeDisplayed: false });
+
+    // expect(true, "Don't forget to check that the content of the Theme Page is as expected.").to.equal(false);
+
+    // Ensure that the navbar title is indeed the expected title.
+    const themePageTitleFinder = await themePage.getPageTitle();
+    await expect(themePageTitleFinder.getText()).to.eventually.have.string(themePage.expectedTitle);
+
+    // Ensure that the content of the page matches the theme data.
+    const themeDetailsText = await themePage.getThemeDetailsText();
+    // @Simon: I don't understand...
+    // Withtout a call to `load` before, the call to `related` fetchs a Theme all right, but it'is NOT the theme to which the action is actually related.
+    // Why do I need to call `load` THEN `related` to access to correct theme? Isn't `related` supposed to load the related theme, as its name would imply?
+    // Plus, calling `load` before `related` would log a request done on the db to `SELECT` the theme from the table.
+    // This log is not there when `load` is not called. Does this mean that `related` does not execute any query?
+    // If so, how the hell does it load a theme at all?
+    await actions[0].load('theme');
+    const theme = await actions[0].related('theme');
+    // This is to detect the above mentioned strange behavior.
+    expect(actions[0].get('theme_id')).to.equal(theme.get('id'));
+    expect(themeDetailsText).to.have.string(theme.get('title'));
+    expect(themeDetailsText).to.have.string(theme.get('description'));
+
+    /**
+     * 8. Clicking on the back button and thus returning to the previous Action page.
+     */
+
+    // Click on the back button from the Theme Page
+    const themePageBackButton = await themePage.getBackButton();
+    await browser.wait(EC.elementToBeClickable(themePageBackButton), AVERAGE_WAIT_TIME);
+    themePageBackButton.click();
```

Comment:
Similarly to my comment in a previous PR, this could be encapsulated in a `goBack` method in `ThemePageObject` (or even in `AbstractPageObject` since `getBackButton` is there, although in that case an assertion should be added to make sure the button is actually there).